AbijahKaj/wakala-interface
# Reorganised the App.js file to support Redux (#222)

* Added Redux and AsyncStorage

* Update App.js

* Update index.js

* Update index.js

* Adding icon and Splash Screen

* Adding working PIN library

* Ejected from Expo to support AsyncStorage

* Released the #1 APK

* Changing the splash screen display mode

* Tested and fixed the iOS version

* Update app-release.apk

* Hide the onboarding section when the user already went through it

All new screens should now be added under src/screens/index.js

* Revert "Merge branch 'dev' of https://github.com/WakalaDAO/wakala-interface into dev"

This reverts commit b2e1ec7f8a29f65ba8b0dd1405eaaf0ea21620b7, reversing
changes made to 3d9aa3ea922d527c438f8c255d529ffadbb7f36d.

* Update app-release.apk

* Update VerifyCeloCodes.js

* Added logout support (not fully)
